indra/newview/scripts/lua/test_debugsettings.lua

```diff
@@ -0,0 +1,10 @@
+local LLDebugSettings = require 'LLDebugSettings'
+local SETTING_NAME = '360CaptureCameraFOV'
+LL.atexit(function() LL.print_info('Finished') end)
+
+LL.print_info('Updating ' .. SETTING_NAME)
+LLDebugSettings.set(SETTING_NAME, 100)
+
+debug = LLDebugSettings.get(SETTING_NAME)
```

Comment:
Let's add a comment here explaining that the point of this assignment is to "inadvertently" hide the Luau library module `debug`.